bin/storm.py

```diff
@@ -263,6 +263,34 @@ def upload_credentials(*args):
         jvmtype="-client",
         extrajars=[USER_CONF_DIR, STORM_BIN_DIR])
 
+def blobstore(*args):
+    """Syntax: [storm blobstore [cmd]]
+
+    list [KEY...] - lists blobs currently in the blob store
+    cat [-f FILE] KEY - read a blob and then either write it to a file, or STDOUT (requires read access).
+    create [-f FILE] [-a ACL ...] [--repl-fctr NUMBER] KEY - create a new blob. Contents comes from a FILE
+         or STDIN. ACL is in the form [uo]:[username]:[r-][w-][a-] can be comma
+         separated list.
+         --repl-fctr refers to replication factor for the blob. Here NUMBER > 0.
+         for example the following would create a mytopo:data.tgz key using the data
+         stored in data.tgz.  User alice would have full access, bob would have
+         read/write access and everyone else would have read access.
+         storm blobstore create mytopo:data.tgz -f data.tgz -a u:alice:rwa,u:bob:rw,o::r
```

Comment:
I didn't read down far enough.  Maybe just move this to a general ACL example below, and some brief pointer to the example from the other commands that take ACL option arguments?